lib/file-helper.js

```diff
@@ -28,7 +27,7 @@ const cache = global.cache || {}
 const getListing = async (directoryPath, pattern = '**/*', options = {}) => {
   const listing = await glob(pattern, {
     allowEmpty: true,
-    cwd: join(cwd(), directoryPath),
+    cwd: directoryPath,
```

Comment:
We run our tooling from the project root so `cwd()` has been (mostly) stable for us

But since Node.js via `node`/`nodemon` can launch from any directory it's better that we use absolute paths instead like we do here